File: app/src/main/java/pl/potera/recyclerview/Movie.java
package pl.potera.recyclerview;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class Movie {
    private String title;
    private String genre;
    private String year;

    public Movie() { }

    public Movie(String title, String genre, String year) {
        this.title = title;
        this.genre = genre;
        this.year = year;
    }
}

File: app/src/main/java/pl/potera/recyclerview/data/Movie.kt
package pl.potera.recyclerview.data

import java.io.Serializable

class Movie (val title: String, val genre: String, val year: String, val icon: String,
             val image: String, val description: String, val photos: PhotoTiles,
             val actors: Array<Actor>): Serializable {
    var stars: Float = 0f
    var seen: Boolean = false
}